support/testing/scenarios/vm.go

```diff
@@ -0,0 +1,364 @@
+package scenarios
+
+import (
+	"context"
+	"fmt"
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/builtin/account"
+	"github.com/filecoin-project/specs-actors/actors/builtin/exported"
+	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
+	"github.com/filecoin-project/specs-actors/actors/runtime"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-hamt-ipld"
+	"github.com/pkg/errors"
+)
+
+// VM holds the state and executes messages over the state.
+type VM struct {
+	ctx   context.Context
+	store adt.Store
+
+	currentEpoch abi.ChainEpoch
+
+	actorImpls  ActorImplLookup
+	actorRoot   cid.Cid  // The last committed root.
```

Comment:
Is this just the state root? The name suggests it's something different, but if it's not, please use the name we're all used to.